utils/constants.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright 2021 The Rode Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const DATE_TIME_FORMAT = "h:mm:ssa | MM-DD-YYYY";
```

Comment:
Thoughts on formatting with the month name instead of the number? There's some ambiguity with whether days or months come first (e.g., is 3/2/2021 March 2nd or Feb 3rd).

Reply:
Good call, that makes sense to me!